.buildkite/scripts/steps/artifacts/docker_image.sh

```diff
@@ -88,12 +87,21 @@ fi
 echo "--- Build dependencies report"
 node scripts/licenses_csv_report "--csv=target/dependencies-$GIT_ABBREV_COMMIT.csv"
 
-echo "--- Upload artifacts"
+echo "--- Upload CDN assets"
```

Comment:
Does this need to be skipped if `--skip-cdn-assets` is passed?

Reply:
It does, but there's no input to add `--skip-cdn-assets` currently.